package.json

```diff
@@ -25,7 +25,7 @@
     "cross-env": "^7.0.0",
     "debug": "^4.1.1",
     "husky": "^4.2.3",
-    "jest": "25.3.0",
+    "jest": "24.9.0",
```

Comment:
@ryardley is the issue with jest 25?

Reply:
I was finding having multiple versions of jest seemed to cause some tests that should have failed to pass. We should try and upgrade it all to 25 to avoid the warning with `ts-jest` but I vaguely remember having issues with that.

Reply:
Yes I just checked and tsdx can't handle the newer jest as it throws these errors: 

```
TypeError: Cannot read property 'run' of undefined
    at /Users/rudiyardley/projects/blitz/blitz/node_modules/tsdx/dist/index.js:418:20
error Command failed with exit code 1.
```

Reply:
I just downgraded ts-jest and everything appears to be working with no warnings.